homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/style.css

```diff
@@ -0,0 +1,28 @@
+.about_me_article {
```

Comment:
read pls about BEM methodology and change names of classes

Reply:
done